aries_cloudagent/holder/routes.py

```diff
@@ -255,7 +255,7 @@ async def register(app: web.Application):
                 credentials_attr_mime_types_get,
                 allow_head=False,
             ),
-            web.post("/credential/{credential_id}/remove", credentials_remove),
+            web.delete("/credential/{credential_id}/remove", credentials_remove),
```

Comment:
Shouldn't it just be `web.delete("/credential/{credential_id}" ..`?

Reply:
Hey, yeah! I will push a change first thing tomorrow.